wrobot_auto_trial.py

```
@@ -2,21 +2,61 @@
+        try:
+            windows_found = pywinauto.findwindows.find_windows(title="Trial version")
+            for hwnd in windows_found:
+                try:
+                    window_title = win32gui.GetWindowText(hwnd)
+                    print(f"Found Trial version popup: '{window_title}' (HWND: {hwnd})")
+                    # Send WM_CLOSE without stealing focus
+                    win32gui.PostMessage(hwnd, win32con.WM_CLOSE, 0, 0)
+                    print("Dismissed Trial version popup (no focus steal)!")
+                except Exception as e:
+                    print(f"Error handling Trial version popup {hwnd}: {e}")
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)